lib/paper_trail/record_trail.rb

```diff
@@ -502,7 +502,7 @@ def habtm_assoc_ids(habtm_assoc)
     end
 
     def log_version_errors(version, action)
-      version.logger.warn(
+      version.logger&.warn(
```

Comment:
What does the ampersand character do in this context?

Does it not make more sense to just wrap these statements on a `if` block that checks to see if `version.logger` is not `nil`?